applications/main/u2f/scenes/u2f_scene_main.c

```diff
@@ -100,7 +100,8 @@ void u2f_scene_main_on_enter(void* context) {
     app->u2f_ready = u2f_init(app->u2f_instance);
     if(app->u2f_ready == true) {
         u2f_set_event_callback(app->u2f_instance, u2f_scene_main_event_callback, app);
-        app->u2f_hid = u2f_hid_start(app->u2f_instance);
+        // app->u2f_hid = u2f_hid_start(app->u2f_instance);
```

Comment:
Could both the USB and NFC interfaces be running at the same time? Or there be some way to select which one you want?